app/controllers/json_api/event/kind_categories_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+class JsonApi::Event::KindCategoriesController < JsonApiController
+  def index
+    authorize!(:index, Event::KindCategory)
+    super
+  end
+
+  def show
+    authorize!(:show, entry)
+    super
+  end
+
+  private
+
+  def entry
+    @entry ||= Event::KindCategory.accessible_by(current_ability).where(deleted_at: nil).find(params[:id])
```

Comment:
```suggestion
    @entry ||= Event::KindCategory.where(deleted_at: nil).find(params[:id])
```
https://github.com/hitobito/hitobito_sac_cas/pull/1144#discussion_r1806154594